pandas/core/generic.py

```diff
@@ -3382,9 +3383,25 @@ def convert_objects(self, convert_dates=True, convert_numeric=False,
 
     @Appender(_shared_docs['fillna'] % _shared_doc_kwargs)
     def fillna(self, value=None, method=None, axis=None, inplace=False,
-               limit=None, downcast=None):
+               limit=None, downcast=None, errors='force'):
         inplace = validate_bool_kwarg(inplace, 'inplace')
 
+        from pandas import Series
```

Comment:
you won't do *any* of this here, this will *all* be done in ``pandas\core\missing.py``